src/core/services/theming/theming.js

```diff
@@ -143,6 +157,12 @@ function detectDisabledThemes($mdThemingProvider) {
  * classes when they change. Default is `false`. Enabling can reduce performance.
  */
 
+/**
+ * @ngdoc method
+ * @name $mdThemingProvider#appendTo
+ * @param {element} element to append the styles too. By default it picks document head
```

Comment:
Can this be changed to "element to append the material theme styles too. By default, the document head is used."?